src/app/recipe/new/page.tsx

```diff
@@ -0,0 +1,87 @@
+import { mockDataNewRecipe } from "@/mock/NewRecipe";
+
+import { CloseButton, Edit, Plus } from "@/components/icons";
+import { ImageCarousel, ImageComponent } from "@/components/image";
+import { AddButton } from "@/app/recipe/new/_components/AddButton";
+import { IngredientListItem } from "@/app/recipe/new/_components/IngredientListItem";
+import { RecipeStepItem } from "@/app/recipe/new/_components/RecipeStepItem";
+
+const Page = () => {
+  const titleClass = "mb-1 ml-4 font-bold text-black";
+  return (
+    <div>
+      <div className="flex items-center justify-between border-b border-lightGray px-3 py-4">
+        <CloseButton />
+        <div className="flex items-center gap-4 font-bold">
+          <button className="text-gray">下書き</button>
+          <button className="text-tomato">作成する</button>
+        </div>
+      </div>
+      <div className="mb-12 mt-5 flex flex-col gap-8">
+        <div>
+          <div className={titleClass}>レシピ名</div>
```

Comment:
細かくてすみませんmm
このあたりとかもhタグ使うかpタグ使うとマークアップの意味合いが正しくなるかと思います。

```suggestion
          <h3 className={titleClass}>レシピ名</h3>
```